### ISSUE TITLE
Add vignette on superspreading pipeline

### ISSUE THREAD

**Issue by joshwlambert** [epiverse-trace/superspreading#16]
Issue #1 proposed having a vignette which included:

1. Read in and tidy data
2. Fit negative binomial model with {fitdistrplus}
3. Apply summary metrics from {superspreading}
4. Simulate with {bpmodels} with negative binomial parameters

Instead a vignette focussing on fitting distributions (#15) was added.

Here I reissue a request to write a vignette which covers the pipeline from initial data wrangling, to parameter estimation, to superspreading metrics and simulating with either {bpmodels} or {epichains}.

**Comment by adamkucharski** [comment]
Agree, would be useful to have a full data-to-fitting-to-scenario pipeline illustrated (although the fitting distribution vignette is a nice stand-alone). Perhaps could also integrate with {ringbp} as @jamesmbaazam  develops those options?

**Comment by joshwlambert** [comment]
I'll use {bpmodels} for now with the aim to convert to using {epichains} once stable. 

I'll track progress on epiverse-trace/epichains#6 and once there is a working version I can incorporate into {superspreading}. 

@jamesmbaazam this could be a good topic of collaboration/cross-pillar work. Let me know if want to collaborate on this development.

**Comment by joshwlambert** [comment]
Removing from v0.2.0 release as it needs to be reviewed whether such a pipeline is within the scope of the {superspreading} package. Possibly more suitable to [case studies](https://github.com/epiverse-trace/casestudies).

**Comment by joshwlambert** [comment]
The main issue here is scope. Including reading, fitting and then simulating all with other packages lends itself more to a [howto](https://github.com/epiverse-trace/howto) than a vignette in {superspreading}.

Tagging @avallecam to see if you agree that we have this has a howto script?

**Comment by avallecam** [comment]
> Tagging [@avallecam](https://github.com/avallecam) to see if you agree that we have this has a howto script?

yes, fill the issue or create the PR! I'm interested in having the code written for this task. Who would be in charge of it?

**Comment by joshwlambert** [comment]
@avallecam sure, I'll write a pipeline script and open a PR to howto.
